accounts-db/src/ancient_append_vecs.rs

```diff
@@ -1462,10 +1462,8 @@ pub mod tests {
                             .map(|store| db.get_unique_accounts_from_storage(store))
                             .collect::<Vec<_>>();
 
-                        let accounts_per_storage = infos
-                            .iter()
-                            .zip(original_results.into_iter())
-                            .collect::<Vec<_>>();
+                        let accounts_per_storage =
```

Comment:
we should remove any changes to core solana codebase for lints/clippy stuff. makes the diff super hard to parse

Reply:
also increases likelihood of merge conflict in the future